src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -437,9 +441,38 @@ public class LibraryServiceClient implements BackgroundResource {
    * <pre><code>
    * try (LibraryServiceClient libraryServiceClient = LibraryServiceClient.create()) {
    *   ListShelvesRequest request = ListShelvesRequest.newBuilder().build();
-   *   ApiFuture&lt;ListShelvesResponse&gt; future = libraryServiceClient.listShelvesCallable().futureCall(request);
+   *   ApiFuture&lt;ListShelvesPagedResponse&gt; future = libraryServiceClient.listShelvesPagedCallable().futureCall(request);
    *   // Do something
-   *   ListShelvesResponse response = future.get();
+   *   for (Shelf element : future.get().iterateAll()) {
+   *     // doThingsWith(element);
+   *   }
+   * }
+   * </code></pre>
+   */
+  public final UnaryCallable<ListShelvesRequest, ListShelvesPagedResponse> listShelvesPagedCallable() {
```

Comment:
This addition is also in java_library.baseline: https://github.com/googleapis/gapic-generator/blob/eea42da690a5cec5660a76c540a45465323128af/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L589